swagger.yaml

```diff
@@ -402,7 +402,13 @@ components:
         main_document_id:
           type: integer
           format: int64
-          example: 1 
+          example: 1
+        created_at:
+          type: string
+          example: 2023-06-17 14:23:48.458950
+        last_modified:
+          type: string
+          example: 2023-06-17 14:23:48.458950
         owner:
```

Comment:
Please add field with owner name

Reply:
hold up, I just realized we don't even store (at all) such a thing like user name ( it's always user_id or user_email). I guess for your sake its user_email that should be returned?

Reply:
Yeah, right, email is fine

Reply:
okey email is now returned, also i intentionally changed the migration 2 to change those names, as i find them very misleading i hope it's not that big of a deal

Reply:
Actually it is a big deal. If you do so, we have to setup whole db from scratch as existing data is not compatible. Please do this in a right way and change column name in new migration